File: app/src/main/java/cos/mos/utils/net/okhttp/HttpListener.java
package cos.mos.utils.net.okhttp;

/**
 * @Description:
 * @Author: Kosmos
 * @Date: 2019.05.06 11:44
 * @Email: KosmoSakura@gmail.com
 */
public interface HttpListener<T> {
    void failure(String describe, int code);

    void success(T response);
}

File: app/src/main/java/cos/mos/utils/laboratory/justfortest/HttpListener.kt
package cos.mos.utils.laboratory.justfortest

/**
 * @Description 我是一个虚假的回调
 * @Author Kosmos
 * @Date 2019.10.18 16:22
 * @Email KosmoSakura@gmail.com
 * */
interface HttpListener<T> {
    fun httpSucceed(out: T)
    fun httpFail(describe: String)
}